alf/algorithms/sac_algorithm_test.py

```diff
@@ -223,7 +223,6 @@ def test_sac_algorithm_discrete(self, use_parallel_network):
             use_parallel_network=use_parallel_network,
             env=env,
             config=config,
-            actor_optimizer=alf.optimizers.Adam(lr=1e-2),
```

Comment:
why is this removed?

Reply:
In the discrete case, there is no actor network. So this optimizer will not be used. Now any unused optimizer will be checked and produce an error.